apps/prairielearn/src/lib/config.ts

```diff
@@ -595,10 +595,7 @@ export async function loadConfig(paths: string[]) {
 }
 
 export function setLocalsFromConfig(locals: Record<string, any>) {
-  locals.homeUrl = config.homeUrl;
   locals.urlPrefix = config.urlPrefix;
   locals.plainUrlPrefix = config.urlPrefix;
```

Comment:
I'll review it tomorrow, but plainUrlPrefix is another good candidate if you want to add it to the mix. I've used config.urlPrefix in some ejs conversions in the past.

Reply:
I might do that in another PR. That's used in many more places and would likely be a little more difficult to review (and thus also easier to review on its own).